_data/navbars/getting-started.yml

```diff
@@ -26,7 +19,14 @@ section:
     - title: IBM Cloud Kubernetes Service (IKS)
       path: /getting-started/kubernetes/managed-public-cloud/iks
     - title: Microsoft Azure Kubernetes Service (AKS)
-      path: /getting-started/kubernetes/managed-public-cloud/aks
+      path: /getting-started/kubernetes/managed-public-cloud/aks  
```

Comment:
I think the order of pages should be: Quickstart, managed pub cloud, self-managed pub cloud, self-managed on-prem, flannel, hard way, system requirements.  

Note that we may change system requirements page to be earlier in the future, but for now I believe it likely causes more friction than it solves.

Reply:
Fixed.